src/index.html

```diff
@@ -7,6 +7,328 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#home">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Смартфоны и телефоны</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Гаджеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Планшеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Фото</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="content">
+      <section class="container">
+        <div class="card container__card" data-qa="card">
```

Comment:
```suggestion
        <div class="card" data-qa="card">
```
I don't actually understand for what you are using mix here. There are no one style for this class.

Reply:
I actually used it in cards block

Reply:
It's ok. I'm speaking about you did't put any style code in this class. For what you need it?

Reply:
check .container in cards.scss again

Reply:
I see now, take my apologies! ;)